aliquot.gemspec

```
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'excon',          '~> 0.71.0'
   s.add_runtime_dependency 'hkdf',           '~> 0.3'
 
-  s.add_development_dependency 'aliquot-pay', '~> 2.1.0'
+  s.add_development_dependency 'aliquot-pay', '~> 2.1.1'
   s.add_development_dependency 'rspec',       '~> 3'
-  s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry',         '~> 0.14.1'
```

Isn't it superfluous to specify a specific version of `pry`?

```suggestion
  s.add_development_dependency 'pry'
```

---

This is to get rid of this warning `gem buld` throws.
```console
➜  aliquot git:(aliquot_2.1.1) ✗ gem build
WARNING:  open-ended dependency on pry (>= 0, development) is not recommended
  use a bounded requirement, such as '~> x.y'
WARNING:  See https://guides.rubygems.org/specification-reference/ for help
```

---

Cool :+1: